curator/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 # by default, run the curator cron job at midnight UTC every night, and delete
 # indices older than $CURATOR_DEFAULT_DAYS days old
-ENV HOME=/opt/app-root/src \
+ENV HOME=/opt/app-root \
```

Comment:
`/opt/app-root/src` is common for all images we're producing (even outside of logging IIRC). any particular reason why we're changing that?